docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -32,6 +32,21 @@ The *:scheme* header will be used by Envoy over *x-forwarded-proto* where the UR
 
 .. _config_http_conn_man_headers_user-agent:
 
+:path
+-----
+
+The *:path* header is a pseudo-header populated by Envoy using the value of the path of the HTTP
```

Comment:
```suggestion
The ``:path`` header is a pseudo-header populated by Envoy using the value of the path of the HTTP
```
as these are literals they should use ` `` ` - im aware that this file has other uses of italics but we are gradually clearing these up

would you mind changing the others

Reply:
Changes added, though there were a few cases where I do not believe code snippets were intended, and these were left in italics.